common/log/logger.go

```diff
@@ -0,0 +1,87 @@
+package log
+
+import (
+	"fmt"
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	// `gl` is the global logger.
+	// Other packages should use public methods like Info/Error to do the logging.
+	// If special logging is required (like log to a separate file for some special operations), we need to add other loggers.
+	gl     *zap.Logger
+	gLevel zap.AtomicLevel
+)
+
+// MustInitialize initializes the global logger for the bytebase server.
+func MustInitialize(debug bool) {
+	if gl != nil {
+		return
+	}
+	gLevel = zap.NewAtomicLevelAt(zap.InfoLevel)
+	if debug {
+		gLevel.SetLevel(zap.DebugLevel)
+	}
+	gl = zap.New(zapcore.NewCore(
+		zapcore.NewConsoleEncoder(zap.NewDevelopmentEncoderConfig()),
+		zapcore.Lock(os.Stdout),
+		gLevel,
+	))
+}
+
+// MustInitializeBB initializes the global logger for the BB CLI.
+func MustInitializeBB() {
```

Comment:
Can we merge BB and Console or what would be the difference? Consider name it log.Init().

Reply:
Done.